src/dns.zig

```diff
@@ -375,10 +375,7 @@ pub const GetAddrInfo = struct {
 
         pub fn toJS(this: *const Result, globalThis: *jsc.JSGlobalObject) JSValue {
             const obj = jsc.JSValue.createEmptyObject(globalThis, 3);
-            obj.put(globalThis, jsc.ZigString.static("address"), addressToJS(&this.address, globalThis) catch |err| return switch (err) {
-                error.JSError => .zero,
-                error.OutOfMemory => globalThis.throwOutOfMemoryValue(),
-            });
+            obj.put(globalThis, jsc.ZigString.static("address"), addressToJS(&this.address, globalThis));
```

Comment:
i wonder if there's other places we can delete a switch on JSError

Reply:
looked, not really